doc/tutorial/fcm_fundamentals

```diff
@@ -0,0 +1,859 @@
+                                <li><a href="fcm_links">Links</a></li>
+                                <li><a href="fcm_mqrg">My Quick Reference Guide</a></li>
+                            </ul>
+                        </li>
+                    </ul>
+                </div>
+                <!--/.nav-collapse -->
+            </div>
+        </nav>
+        <div class="container theme-showcase" role="main">
+        <div class="jumbotron">
+            <h1>Fundamentals of version control, svn and fcm</h1>
+            <p>This aims to explain the importance of:
+            <ul>
+                <li><a href="#S1">Using version control</a></li>
+                <li><a href="#S2">The basics of svn and where to learn more</a></li>
+                <li><a href="#S3">Scenarios</a></li>
+                <li><a href="#S4">General configuration</a></li>
+                <li><a href="#S5">Setting-up your own repository</a></li>
+                <li><a href="#S6">Setting-up the tutorial repositories</a></li>
+            </ul>
+            </p>
+        </div>
+        <div class="well">
+            First lets do the Fundamentals Quiz to see how much you already know and what we
+            will need to focus on in this section, note there are no right or wrong answers
+            for the first five questions.
+        </div>
+        <div class="container">
+            <p><a class="btn btn-lg btn-success" href="fundamentals_quiz" role="button">Fundamentals Quiz</a></p>
+        </div>
+        <br/>
+        <h1><a id="S1">Using version control</a></h1>
+        <p>
+        <ul>
+            <li>
+                <h2>Introduction</h2>
+            </li>
+            <div class="page-header">
+                <h3>Why use a version control system?</h3>
+            </div>
+            <div class="well">
+                Any good version control system provides helpful tools to allow users to;
+                <ul>
+                    <li>share work and encourage collaboration,</li>
+                    <li>track the progression and incremental changes to versions/revisions of 
+                        work,
+                    </li>
+                    <li>allow multiple flavours of the some work to investigate the impact of a 
+                        change,
+                    </li>
+                    <li>traceability of work, for example what version of the work was used and 
+                        when,
+                    </li>
+                    <li>to revert a change,</li>
+                    <li>branch from a version, 
+                    <li>merge in changes</li>
+                    <li>and so much more.</li>
+                </ul>
+                <br/>
+            </div>
+            <div class="page-header">
+                <h3>Subversion</h3>
+            </div>
+            <div class="well">
+                Subversion has some very good documentation, Version Control with Subversion, 
```

Comment:
I'd put the link on the Version Control with Subversion words - makes it clear that when you talk about that that it is the documentaion